primitives/primproc/columncommand.cpp

```diff
@@ -249,7 +250,36 @@ void ColumnCommand::loadData()
 
 void ColumnCommand::issuePrimitive()
 {
-    _issuePrimitive();
+    loadData();
+
+    if (!suppressFilter)
+        bpp->pp.setParsedColumnFilter(parsedColumnFilter);
```

Comment:
`bpp->getPrimitiveProcessor().setParsedColumnFilter(parsedColumnFilter);` ? // and below too

Reply:
Questionable b/c ColumnCommand is a friend to BPP and has direct access to pp but fixed.